megalista_dataflow/uploaders/google_ads/conversions/google_ads_enhanced_conversions_leads_uploader.py

```diff
@@ -53,15 +53,15 @@ def _get_customer_id(self, account_config: AccountConfig, destination: Destinati
             If the customer_id is present on the destination, returns it, otherwise defaults to the account_config info.
         """
         if len(destination.destination_metadata) >= 2 and len(destination.destination_metadata[1]) > 0:
-            return destination.destination_metadata[1].replace('-', '')
-        return account_config.google_ads_account_id.replace('-', '')
+            return utils.clean_ads_customer_id(destination.destination_metadata[1])
```

Comment:
Since clean_ads_customer_id and filter_text_only_numbers perform the exact same operation, could we replace the clean_ads_customer_id for filter_text_only_numbers instead?